hoomd/mpcd/DryingDropletStreamingMethod.h

```diff
@@ -129,37 +131,8 @@ void DryingDropletStreamingMethod::stream(unsigned int timestep)
     const unsigned int N_remove = round((4.*M_PI/3.)*end_R*end_R*end_R*m_density);
     const int N_evap = m_mpcd_pdata->getNGlobal() - N_remove;
 
-    // get the compact array of indexes of bounced particles
-    unsigned int N_bounced = 0;
-        {
-        const unsigned int N = m_mpcd_pdata->getN();
-        ArrayHandle<unsigned char> h_bounced(m_bounced, access_location::host, access_mode::read);
-
-        bool overflowed = false;
-        do
-            {
-            if (overflowed)
-                {
-                GPUArray<unsigned int> bounced_index(N_bounced, m_exec_conf);
-                m_bounced_index.swap(bounced_index);
-                // resize m_bounced_index with swap idiom
-                }
-            ArrayHandle<unsigned int> h_bounced_index(m_bounced_index, access_location::host, access_mode::overwrite);
-
-            const unsigned int N_bounced_max = m_bounced_index.getNumElements();
-            N_bounced = 0;
-            for (unsigned int idx=0; idx < N; ++idx)
-                {
-                if (h_bounced.data[idx])
-                    {
-                    if (N_bounced < N_bounced_max)
-                        h_bounced_index.data[N_bounced] = idx;
-                    ++N_bounced;
-                    }
-                }
-            overflowed = N_bounced > N_bounced_max;
-            } while (overflowed);
-        }
+    // get the compact array of indexes of bounced particles and total N_bounced
+    unsigned int N_bounced = calculateN_bounced();
```

Comment:
Name the function more like this, HOOMD doesn't use underscores in function names
```suggestion
    unsigned int N_bounced = calculateNumBounced();
```